src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -401,8 +611,40 @@ services.AddHttpClient(
 ```
 
 > [!NOTE]
-> The single instance returned by `HttpClientFactory` is reused by all export
-requests.
+> `IHttpClientFactory` is NOT currently supported by `OtlpLogExporter`.
+
+## Experimental features
+
+The following features are exposed experimentally in the OTLP Exporter. Features
+are exposed experimentally when either the [OpenTelemetry
+Specification](https://github.com/open-telemetry/opentelemetry-specification)
+has explicitly marked something
+[experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md)
+or when the SIG members are still working through the design for a feature and
+want to solicit feedback from the community.
+
+### Environment variables
+
+> [!NOTE]
+> In OpenTelemetry .NET environment variable keys are retrieved using
+  `IConfiguration` which means they may be set using other mechanisms such as
+  defined in appSettings.json or specified on the command-line.
+
+* All signals
+
+  * `OTEL_DOTNET_EXPERIMENTAL_OTLP_ENABLE_INMEMORY_RETRY`
+ 
+    When set to `true`, it enables in-memory retry for transient errors
+    encountered while sending telemetry.
+
+    Added in `1.8.0-beta.1`.
+
+* Logs
+
+  * `OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES`
+
+    When set to `true`, it enables export of `LogRecord.EventId.Id` as
+    `logrecord.event.id` and `LogRecord.EventId.Name` as `logrecord.event.name`.
```

Comment:
Since you have added the starting version for the above feature, maybe add it for this one as well?